tools/picard/picard_macros.xml

```diff
@@ -81,5 +81,7 @@ This will be added shortly
 
     </token>
     
-
+    <token name="@TMP_DIR_FIX@"><!-- Bugfix that ensures that Picards temp dir is set to the galaxy temp dir -->
+        TMP_DIR=$__new_file_path__
```

Comment:
There is a good chance that `$TMPDIR` is set. In cases where its not, the admin can set it in `job_conf.xml`. But yes a reboot might be needed.